superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -834,6 +847,26 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     setTabKey(key);
   };
 
+  const renderStepTwoAlert = () =>
+    db?.engine && (
+      <StyledAlertMargin>
+        <Alert
+          closable={false}
+          css={(theme: SupersetTheme) => antDAlertStyles(theme)}
+          type="info"
+          showIcon
+          message={
+            engineSpecificAlertMapping[db.engine]?.message ||
+            connectionAlert?.ALLOWED_IPS?.message
```

Comment:
instead of naming it Allowed_IPS we should have a more general name. Then users can set this value to whatever they'd like. Let's make the key DEFAULT and then we'd update our superset config accordingly